File: src/types/pengaturan.ts
export type PengaturanGroup = "apotek" | "struk" | "stok" | "notifikasi";

export interface Pengaturan {
  id: string;
  cabangId?: string;
  key: string;
  value: string;
  group: PengaturanGroup;
  label: string;
}

File: backend/app/models/Pengaturan.go
package models

import "time"

type Pengaturan struct {
	ID        string    `json:"id"`
	CabangID  *string   `json:"cabang_id"`
	Key       string    `json:"key"`
	Value     *string   `json:"value"`
	Group     *string   `json:"group"`
	Label     *string   `json:"label"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
}
